docs/notebooks/low_level.ipynb

```diff
@@ -0,0 +1,745 @@
+{
```

Comment:
<p>nit/suggestion: "willing to work with nested data in pandas more computationally and memory efficiently, and having access to different data representations. " -&gt; "looking to optimize the performance, both computationally and in memory-usage, of their workflows. This section also details a broader suite of data representations usable within nested-pandas."</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/lincc-frameworks/nested-pandas/pull/71/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='0'/>